fiftyone/server/query.py

```
@@ -148,6 +153,46 @@ def _create_config(self):
 @gql.type
 class BrainRun(Run):
     config: t.Optional[BrainRunConfig]
+    #: Whether the run's results have been saved.
+    ready: t.Optional[bool]
+    #: Why this run cannot be used, when that is knowable WITHOUT loading its
+    #: results — a config class that no longer imports (a rename the run
+    #: predates), or a malformed run document.
+    error: t.Optional[str]
+
+
+@lru_cache(maxsize=None)
```

Ruff (0.16.0)

[warning] Use `@functools.cache` instead of `@functools.lru_cache(maxsize=None)`

Rewrite with `@functools.cache

(UP033)